en/after-payment/overview.md

```diff
@@ -1,32 +1,38 @@
 # Getting Notified of Payment Status
 <hr>
+You can get notifications when the payment is accepted successfully on Midtrans, so that you can process the customer order accordingly. Once the payment process is completed by the customer, Midtrans sends notification to you. Alternatively, you can retrieve the transaction status from the Midtrans Administrative Portal (MAP). You can access transaction notifications in any of the following ways.
```

Comment:
@AnujaChangede 
> Alternatively, you can retrieve the transaction status from the Midtrans Administrative Portal (MAP)

Can this not specifically mention `Midtrans Administrative Portal (MAP)` here? What we are trying to say to merchant is:
- Other than just **passively** waiting for notification from Midtrans side,  merchant can also **actively** retrieve transaction status from Midtrans, which not just from MAP, but also from API. So we don't need to mention MAP specifically.